### PR TITLE
90 add and configure spacewalk pallets for amplitude runtime

### PR DESCRIPTION
**Pull request by b-yap**
this needs https://github.com/pendulum-chain/pendulum/pull/91.

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -560,15 +581,103 @@ fn amplitude_genesis(
...
+			replace_griefing_collateral: FixedU128::checked_from_rational(1, 10).unwrap(), // 10%
+		},
+		nomination: amplitude_runtime::NominationConfig { is_nomination_enabled: false },
+		dia_oracle: amplitude_runtime::DiaOracleConfig {
+			authorized_accounts: authorized_oracles,
+			supported_currencies: vec![foucoco_runtime::AssetId::new(
+				b"Polkadot".to_vec(),
+				b"DOT".to_vec(),
+			)],
+			batching_api: b"http://localhost:8070/currencies".to_vec(),
```
[the remark this answers is not in the document]
Comment (b-yap) [reply node/src/chain_spec.rs:660]: so @cr4pt0 , is this `vec![]` instead?

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -565,6 +581,85 @@ fn amplitude_genesis(
...
 		},
+		tokens: amplitude_runtime::TokensConfig {
+			// Configure the initial token supply for the native currency and USDC asset
+			balances: token_balances,
+		},
+		issue: amplitude_runtime::IssueConfig {
+			issue_period: amplitude_runtime::DAYS,
+			issue_minimum_transfer_amount: 1000,
+			limit_volume_amount: None,
+			limit_volume_currency_id: XCM(DOT),
```
[the remark this answers is not in the document]
Comment (ebma) [reply node/src/chain_spec.rs:592]: So we would need to set the amount to 900 units (maybe use the .one() function of a currency to take into account the decimals, ie. `SomeCurrency.one().mul(900)`). And for the `limit_volume_currency_id` we might want to actually use our wrapped USDC asset.

File: node/src/chain_spec.rs [hunk 3]
```diff
@@ -565,6 +581,85 @@ fn amplitude_genesis(
...
+		},
+		nomination: amplitude_runtime::NominationConfig { is_nomination_enabled: false },
+		dia_oracle: amplitude_runtime::DiaOracleConfig {
+			authorized_accounts: authorized_oracles,
+			supported_currencies: vec![foucoco_runtime::AssetId::new(
+				b"Polkadot".to_vec(),
+				b"DOT".to_vec(),
+			)],
+			batching_api: vec![],
+			coin_infos_map: vec![],
```
Comment (ebma) [inline node/src/chain_spec.rs:661]: This has to change but I'm not yet sure to what.

**Review by ebma — CHANGES_REQUESTED** [review]

**Review by cr4pt0 — APPROVED** [review]

**Comment by b-yap** [comment]
Closing this one. proceed to PR https://github.com/pendulum-chain/pendulum/pull/116

**Pull request closed without merging**